src/caffe2-ios/caffe2-ios/staticDetectorVC.swift

```diff
@@ -60,7 +67,29 @@ class staticDetectorVC: UIViewController {
         // Dispose of any resources that can be recreated.
     }
     
-
-
+    // MARK: Image Picker Controller Delegate Functions
+    func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [String : Any]) {
+        var pickedImage : UIImage
+        if let possibleImage = info["UIImagePickerControllerOriginalImage"] as? UIImage {
+            pickedImage = possibleImage
+        } else {
+            return
+        }
+        self.classifier(image: pickedImage)
+        dismiss(animated: true, completion: nil)
+    }
+    // MARK: Help functions
+    func resizeImage(image: UIImage, newWidth: CGFloat) -> UIImage? {
```

Comment:
For this kind of helper function, we may wanna do a separate service to handle it?